lwgenerate/gcp/gcp_test.go

```diff
@@ -0,0 +1,574 @@
+package gcp
```

Comment:
Are we testing the functions exposed to the consumers? or internal functions?

I feel these are tests for consumers so we should rename the test package and import the `lwgenerate/gcp` package as a subject
```suggestion
package gcp_test
```